pymilo/chains/neural_network_chain.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+"""PyMilo chain for linear models."""
+from ..pymilo_param import SKLEARN_NEURAL_NETWORK_TABLE
+
+def is_neural_network(model):
+    """
+    Check if the input model is a sklearn's neural network.
+
+    :param model: is a string name of a neural netowrk or a sklearn object of it
```

Comment:
`:param model: name of a neural network or a sklearn object of it`
+ note the typo in `network`